templates/flower/flower-auth-sidecar-configmap.yaml

```
@@ -28,8 +28,7 @@ data:
       server {
         server_name {{ .Release.Name }}-flower.{{ .Values.ingress.baseDomain }} ;
         listen {{ .Values.authSidecar.port }}  ;
-        # Disable Nginx Server version
-        server_tokens off;
+{{ include "default_nginx_server_settings" . | indent 8 }}
```

Since this isn't nindent of a data structure, should we just indent it so it aligns with the code, rather than calling `| indent 8` ?
```suggestion
        {{ include "default_nginx_server_settings" . }}
```